src/tools/fuzzing/fuzzing.cpp

```diff
@@ -1688,6 +1691,73 @@ Expression* TranslateToFuzzReader::makeTry(Type type) {
   return builder.makeTry(body, catchTags, catchBodies);
 }
 
+Expression* TranslateToFuzzReader::makeTryTable(Type type) {
+  if (funcContext->breakableStack.empty()) {
+    // Nothing to break to.
+    // TODO: Perhaps generate a block wrapping us?
+    return makeTrivial(type);
+  }
+  if (wasm.tags.empty()) {
+    addTag();
+  }
+
+  // Add catches of specific tags, and possibly a catch_all at the end. We use
+  // the last iteration of the loop for that.
+  std::vector<Name> catchTags;
+  std::vector<Name> catchDests;
+  std::vector<bool> catchRefs;
+  auto numTags = upTo(MAX_TRY_CATCHES);
+  for (Index i = 0; i <= numTags; i++) {
+    Name tagName;
+    Type tagType;
+    if (i < numTags) {
+      // Look for a specific tag.
+      auto& tag = pick(wasm.tags);
+      tagName = tag->name;
+      tagType = tag->sig.params;
+    } else {
+      // Look for a catch_all at the end, some of the time (but all of the time
```

Comment:
What do you mean "Look for"? Aren't we _generating_ a `catch_all` here, not looking for (an already existing) one?

Reply:
Yeah, that was written poorly, thanks. Fixed.